include/libaries_askar.h

```diff
@@ -7,7 +7,7 @@
 
```

Comment:
This is a generated file so if something changes here we should make sure that cbindgen generates the file with these changes as this will be overwritten when a new header is generated. Could you make it gets generated with your changes?

Reply:
I'm not sure if it's possible. I've created an issue (#99) to track this.

Reply:
done